packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -346,10 +355,41 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
   ///  * [findAll], which is similar but returns all annotations found at the
   ///    given position.
   ///  * [AnnotatedRegionLayer], for placing values in the layer tree.
-  AnnotationEntry<S> find<S>(Offset localPosition) {
+  S find<S>(Offset localPosition) {
```

Comment:
are these old APIs now deprecated? if so please mark them

@Deprecated('Use ... instead. This API will be removed in early 2020.')

Reply:
Added deprecation to findAll. The `find` is kept since there is no demand to extra information for now.